share/p2p/shrexpush/pubsub.go

```diff
@@ -61,11 +64,7 @@ func (s *PubSub) Start(context.Context) error {
 // * Unregisters all the added Validators
 // * Closes the `ShrEx/Sub` topic
 func (s *PubSub) Stop(context.Context) error {
-	err := s.pubSub.UnregisterTopicValidator(s.pubSubTopic)
-	if err != nil {
-		return err
-	}
-
+	// TODO(@vgonkivs): unregister the topic validator
```

Comment:
I would leave the unregistration and just log the error if it happens ( we can make an issue for handling the error later if it's really necessary ).